src/routes/AppRouter.js

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import LandingPage from '@pages/loginPage';
 import HomePage from '@pages/homePage';
+import ProfilePage from '@pages/profilePage';
 import NotFoundPage from '@pages/notFoundPage';
-import SocialMediaLogin from '@components/SocialMedia/SocialMediaLogin';
+import PrivateRoute from './PrivateRoute';
 
 const App = () => (
   <Router>
     <Switch>
       <Route path="/" exact component={HomePage} />
-      <Route path="/login" exact component={SocialMediaLogin} />
-      <Route path="/" component={NotFoundPage} />
+      <Route path="/login" exact component={LandingPage} />
+      <Route path="/" exact component={NotFoundPage} />
```

Comment:
kindly remove one of the route paths to the not found page and bring the "/" for the not found page to the last path of the route